### DISCUSSION TITLE
Github is misidentifying my education email address.

### DISCUSSION THREAD

**Discussion by FXAZfung** [community/community#142221] [GitHub Education]
### Hi everyone,

I am using the education email address of East China University of Technology(ECUT) but it automatically recognises me as the education email address of East China University of Science and Technology(ECUST), the names of these two schools are very similar in China, and the translation into English creates an error, resulting in the failure of my education authentication, is there a solution to this problem?
![image](https://github.com/user-attachments/assets/934f2cf2-8c0f-4205-b509-04a81c88dd71)

**Comment by davevad93** [comment]
Hi @FXAZfung ,

Most likely GitHub has incorrect information about your school, to solve this contact support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.
